### PR TITLE
Add access_log to tools:expire-data, refs #13578

### PR DESCRIPTION
**Pull request by djjuhasz**
- Add "access_log" type to `tools:expire-data` task
- Add "default_max_age" and "method_name" to type specification array
- Wrap long lines

### CODE REVIEW

File: lib/model/QubitAccessLog.php [hunk 1]
```diff
@@ -48,4 +48,19 @@ public static function getPopularThisWeek(array $options = [])
 
         return $stmt->fetchAll();
     }
+
+    /**
+     * Delete old access_log records.
+     *
+     * @return int number of rows deleted
+     */
+    public static function expire(string $expiryDate): int
```
Comment (mcantelon) [inline lib/model/QubitAccessLog.php:60]: Good thinking moving expiry logic to the data type's class!

File: lib/model/QubitAccessLog.php [hunk 2]
```diff
@@ -48,4 +48,19 @@ public static function getPopularThisWeek(array $options = [])
 
         return $stmt->fetchAll();
     }
+
+    /**
+     * Delete old access_log records.
+     *
```
Comment (mcantelon) [inline lib/model/QubitAccessLog.php:54]: Nitpick: omment doesn't document the `$expiryData` argument using `@param`.

**Review by mcantelon — APPROVED** [review]
One nitpick, but otherwise 💯 !

File: lib/model/QubitAccessLog.php (hunk 2 above)
Comment (djjuhasz) [reply lib/model/QubitAccessLog.php:54]: Good call.  I've added a comment for the `$expiryDate` parameter.

File: lib/model/QubitAccessLog.php (hunk 1 above)
Comment (djjuhasz) [reply lib/model/QubitAccessLog.php:60]: Thanks! :)

**Pull request merged**